packages/react-magma-dom/src/components/NativeSelect/NativeSelect.tsx

```diff
@@ -22,19 +23,36 @@ export interface NativeSelectProps
   /**
    * @internal
    */
-  testId?: string;
   optionLabel?: string;
+  testId?: string;
 }
-const StyledNativeSelect = styled.select<{
+const StyledNativeSelectWrapper = styled.div<{
+  disabled?: boolean;
+  hasError?: boolean;
+  isInverse?: boolean;
   theme: ThemeInterface;
+}>`
+  ${inputWrapperStyles}
+  padding: 0 ${props => props.theme.spaceScale.spacing03} 0 0;
+  svg {
+    color: ${props =>
+      props.isInverse && props.disabled
+        ? transparentize(0.6, props.theme.colors.neutral100)
+        : props.disabled
+        ? transparentize(0.4, props.theme.colors.neutral500)
+        : 'inherit'};
+    margin: 0 0 0 -${props => props.theme.spaceScale.spacing06};
+    pointer-events: none;
+    z-index: 1;
+  }
+`;
+
+const StyledNativeSelect = styled.select<{
   isInverse?: boolean;
+  theme: ThemeInterface;
 }>`
+  margin: -1px 0;
   ${inputBaseStyles};
-  background: inherit;
-  > option {
-    background: ${props =>
-      props.isInverse ? props.theme.colors.neutral600 : 'none'};
-  }
```

Comment:
We need to keep this. This was added as part of [fix(nativeselect): add background color to the options for isInverse](https://github.com/cengage/react-magma/commit/ab69379bb8dda7094eba22034e738387e3228a40) 

![image](https://user-images.githubusercontent.com/91160746/182156396-da30e506-0523-43cc-b7df-3c718bcf731c.png)

Reply:
We should maybe even add a comment so we don't accidentally remove it
```
// Required for Windows && Chrome support
```